arbitrum-docs/node-running/how-tos/read-sequencer-feed.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to read the sequencer feed"
+title: "How to Read the Sequencer Feed?"
```

Comment:
I would leave it as "How to read the sequencer feed" (without question mark). Not sure about Sequencer being uppercase or lowercase though.